docs/README.md

```diff
@@ -46,6 +46,31 @@ CMake Tools is an extension designed to make it easy to work with CMake-based pr
 * [CMake Tools settings](cmake-settings.md#cmake-settings)
 * [Variable substitution](cmake-settings.md#variable-substitution)
 
+[Use CMakePresets](cmake-presets.md)
+* [Configure and build with CMake Presets](cmake-presets.md#configure-and-build-with-cmake-presets)
+* [Supported CMake and CMakePresets.json versions](cmake-presets.md#supported-cmake-and-cmakepresetsjson-versions)
+* [Enable CMakePresets.json in the CMake Tools extension](cmake-presets.md#enable-cmakepresetsjson-in-the-cmake-tools-extension)
+* [Configure and build](cmake-presets.md#configure-and-build)
+* [CMake: Configure](cmake-presets.md#cmake-configure)
+* [CMake: Select Build Preset](cmake-presets.md#cmake-select-build-preset)
+* [CMake: Build](cmake-presets.md#cmake-build)
+* [Test](cmake-presets.md#test)
+* [CMake: Select Test Preset](cmake-presets.md#cmake-select-test-preset)
+* [CMake: Run Tests](cmake-presets.md#cmake-run-tests)
+* [Add new presets](cmake-presets.md#add-new-presets)
+* [Add new Configure Presets](cmake-presets.md#add-new-configure-presets)
+* [Add new Build Presets](cmake-presets.md#add-new-build-presets)
+* [Edit presets](cmake-presets.md#edit-presets)
+* [Select your generator](cmake-presets.md#select-your-generator)
+* [Set and reference environment variables](cmake-presets.md#set-and-reference-environment-variables)
+* [Vcpkg integration](cmake-presets.md#vcpkg-integration)
+* [Command substitution in launch.json and settings.json](cmake-presets.md#command-substitution-in-launchjson-and-settingsjson)
+* [Ignored settings](cmake-presets.md#ignored-settings)
+* [Unsupported commands](cmake-presets.md#unsupported-commands)
+* [Troubleshooting](cmake-presets.md#troubleshooting)
+* [Run CMake from the command line or a Continuous Integration (CI) pipeline](cmake-presets.md#run-cmake-from-the-command-line-or-a-continuous-integration-ci-pipeline)
+* [Sourcing the environment when building with command line generators on Windows](cmake-presets.md#sourcing-the-environment-when-building-with-command-line-generators-on-windows)
+* [Example CMakePresets.json file](cmake-presets.md#example-cmakepresetsjson-file)
```

Comment:
It looks like there is no break between this section and the "Troubleshoot CMake Tools" section below. That should be a new header.

Reply:
fixed